portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,54 @@
-         My favourite sci-fic series is Black Mirror!  I am really excited to join Google as an intern this year and I am looking forward to getting to know you!</p>
-      <img src="images/photo.JPG" alt="photo of me" />
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Get a fact</button>
+         My favourite sci-fic series is Black Mirror! I am really excited to join Google as an intern this year and I am looking forward to getting to know you!</p>
+      <p>See me on <a href="https://www.linkedin.com/in/yuxin-jiang-17b36b17a/">Linkedin</a> and 
+        <a href="https://github.com/jyx11011">github</a></p>
+      <div id="prjects">
+        <h3 class="sec-header">My Projects</h3>
+        <p>Here are some projects that I have worked on: </p>
+        <ul>
+          <li>
+            <a href="https://spherical-wave-253606.df.r.appspot.com/stats.html">Fantastic Feasts: </a>
+            A web application for food enthusiasts to look for and share delicious food spot.
+            <a href="https://github.com/team-45-su-19/team-45-codeu">Github repo</a>
+          </li>
+          <li>
+            <a href="https://frozen-coast-38560.herokuapp.com/">Todo Manager: </a>
+            A web application for users who want to manage their todo list easily.
+            <a href="https://github.com/jyx11011/todo-manager">Github repo</a>
+          </li>
+          <li>
+            Sugar Mummy: A command line app that aims to combat Type II diabetes by encouraging users to keep a healthy lifestyle and helping them keep track of daily life.
+            <a href="https://github.com/jyx11011/main">Github repo</a>
+          </li>
+          <li>
+            Vois: An iOS app that helps budding musicians, hobbyists and CCA groups seamlessly track their music progress and seek feedback from peers and the community.
+          </li>
+          <li>
+            Peggle: An iOS app that is similar to the video game Peggle.
+          </li>
+        </ul>
+      </div>
+      <h3 class="sec-header">My Image Gallery</h3>
+      <div id="gallery">
+        <img src="images/photo.JPG" alt="photo of me" />
+        <img src="images/jewel.jpg" alt="jewel" />
+        <img src="images/gardenbythebay.jpg" alt="gardent-by-the-bay" />
+        <img src="images/mountain.JPG" alt="mountain-and-lake" />
+        <img src="images/town.JPG" alt="town" />
+        <img src="images/stream.JPG" alt="stream" />
+        <img src="images/sushi.jpg" alt="sushi" />
+        <img src="images/fried-rice.JPG" alt="fried-rice.">
+        <img src="images/dimsum.JPG" alt="dimsum" />
+      </div>
+      <p>My image gallery includes photos of me and beautiful views where I travelled.
+         There are also photos of delicious food I tried in restaurants as well as food 
+         cooked by myself.</p>
+      <h3 class='sec-header'>Random Facts</h3>
```

Comment:
Use double quotation mark for attribute values.
ref - https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks